docling_serve/datamodel/responses.py

```diff
@@ -35,6 +35,10 @@ class ConvertDocumentResponse(BaseModel):
     timings: dict[str, ProfilingItem] = {}
 
 
+class PresignedUrlConvertDocumentResponse(BaseModel):
+    url: str
```

Comment:
`url` is a bit wrong here, because we don't have the GET url. I think we better skip it.